integration_files/SweRV_EH1/yaml/rtl_simulation.yaml

```diff
@@ -44,3 +44,20 @@
       -cm line+tgl+branch
       -cm_name test_<test_name>_<iteration>
       -cm_dir <out>/test.vdb
+
+- tool: verilator
+  compile:
+    cmd:
+      - "verilator --cc -CFLAGS \"-std=c++11\"
+         -Wno-UNOPTFLAT -I/testbench
+         -f flist_verilator.f --top-module tb_top -exe test_tb_top.cpp --autoflush --trace
+         -f testbench_veri.f"
+      - "cp /home/users/scratch/amnafayyaz.data/05012021/LM-RISCV-DV/integrated_cores/SweRV_EH1/testbench/test_tb_top.cpp obj_dir"
```

Comment:
@amnafayyaz-lm This is not right. Path to file is hardcoded for your directory only. Need to fix it first, commit and push to the same branch.